scalableminds/webknossos
# Use correct volume download route in TracingStoreRpcClient

### URL of deployed dev instance (used for testing):
- https://volumedownloadroute.webknossos.xyz

### Steps to test:
- create volume tracing, trace some, download zip
- the zip should contain a data.zip which should contain some wkw files

### Issues:
- fixes #3400 

------
